clients/lsp-asm.el

```diff
@@ -81,7 +81,8 @@ Will update if UPDATE? is t."
  (make-lsp-client
   :new-connection (lsp-stdio-connection
                    #'lsp-asm--server-command
-                   (lambda () (f-exists? lsp-asm-store-path)))
+                   (lambda () (or (executable-find "asm-lsp"))
+                             (f-exists? lsp-asm-store-path)))
```

Comment:
Can you fix the indentation here? Thanks!

Reply:
Damn, thanks for pointing this out. Apparently I didn't have parinfer on and the `or` clause was missing the `f-exists?`. My bad. Fixed @ latest